pkg/authorization/apis/authorization/conversion.go

```diff
@@ -241,3 +243,38 @@ func Convert_rbac_PolicyRules_To_authorization_PolicyRules(in []rbac.PolicyRule)
 	}
 	return rules
 }
+
+func translate_Origin_Annotations_To_rbac_Annotation(role *rbac.ClusterRole) {
+	// can't use const in pkg/oc/admin/policy because of import cycle
+	_, ok := role.Annotations["openshift.io/reconcile-protect"]
+	if ok {
```

Comment:
uhmm I think this will modify the original metadata ?
I did this as otherwise I fear we need a deepcopy ?